src/Test/PHPUnit/AbstractCacheConstraint.php

```diff
@@ -62,6 +62,6 @@ protected function matches($other)
      */
     protected function failureDescription($other)
     {
-        return (string) $other . ' ' . $this->toString();
+        return 'the responses are identical';
```

Comment:
No verbosity level, but good point about the error status codes: you do want to know about that. I improved the assertion message somewhat so it now says: ‘Failed asserting that response (with status code 500) is a cache hit’.